platform/windows/display_server_windows.cpp

```diff
@@ -2024,9 +2034,15 @@ void DisplayServerWindows::window_set_vsync_mode(DisplayServer::VSyncMode p_vsyn
 	}
 #endif
 
+#if defined(D3D12_ENABLED)
+	if (context_d3d12) {
+		context_d3d12->set_vsync_mode(p_window, p_vsync_mode);
+	}
+#endif
+
 #if defined(GLES3_ENABLED)
-	if (gl_manager) {
```

Comment:
Accidental modification of GLES3 code.

Reply:
Indeed. It was a tough rebase. 😅 Fixed in last push.